src/machines/should-i-train-machine.tsx

```diff
@@ -42,105 +45,50 @@ const shouldITrainQuizMachine = createMachine<
 >(
   {
     id: 'shouldITrainForm',
-    initial: 'enteringTrained',
+    initial: 'entering',
     context: {responses: [], currentQuestion: quiz.questions[0]},
     states: {
-      enteringTrained: {
-        on: {
-          CONFIRM: {
-            target: 'enteringSoreness',
-            actions: ['assignScore', 'assignCurrentQuestion'],
-          },
-        },
-      },
-      enteringSoreness: {
-        on: {
-          BACK: {
-            target: 'enteringTrained',
-          },
-          CONFIRM: {
-            target: 'enteringExcited',
-            actions: ['assignScore', 'assignCurrentQuestion'],
-          },
-        },
-      },
-      enteringExcited: {
-        on: {
-          BACK: {
-            target: 'enteringSoreness',
-          },
-          CONFIRM: {
-            target: 'enteringMood',
-            actions: ['assignScore', 'assignCurrentQuestion'],
-          },
-        },
-      },
-      enteringMood: {
-        on: {
-          BACK: {
-            target: 'enteringExcited',
-          },
-          CONFIRM: {
-            target: 'enteringHormones',
-            actions: ['assignScore', 'assignCurrentQuestion'],
-          },
-        },
-      },
-      enteringHormones: {
+      entering: {
```

Comment:
there is only 4 states to this machine now:

- entering
- saving
- error
- success